compiler/rustc_codegen_llvm/src/gotoc/mir_to_goto/codegen/typ.rs

```diff
@@ -365,6 +367,7 @@ impl<'tcx> GotocCtx<'tcx> {
         use rustc_middle::ty::print::Printer;
         let mut name = String::new();
         let printer = FmtPrinter::new(self.tcx, &mut name, Namespace::TypeNS);
+        let t = self.monomorphize(t);
```

Comment:
I would like to add a comment here with the example:
```
StructTag("tag-std::boxed::Box<dyn std::error::Error + std::marker::Send + std::marker::Sync>") }
StructTag("tag-std::boxed::Box<(dyn std::error::Error + std::marker::Send + std::marker::Sync)>") }
```

Reply:
Added, thanks!